src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -103,7 +103,7 @@
     text: |
         end=`date -v+30M '+%Y-%m-%dT%H:%MZ'`
         az storage account generate-sas --permissions cdlruwap --account-name MyStorageAccount --services qt --resource-types sco --expiry $end -o tsv
-  - name: Generate a shared access signature for the account (autogenerated)
+  - name: Generates a shared access signature for the account (autogenerated)
     text: |
```

Comment:
Aladdin didn't learn. (Maybe we haven't changed all of wrong text?)

Reply:
sure

Reply:
Fixed.